Index: src/3rdParty/Makefile.am
===================================================================
--- src/3rdParty/Makefile.am	(Revision 2056)
+++ src/3rdParty/Makefile.am	(Arbeitskopie)
@@ -1,5 +1,3 @@
-SUBDIRS=Pivy
-
 EXTRA_DIST = \
 		boost/numeric/bindings/LICENSE_1_0.txt \
 		boost/numeric/bindings/amos/amos.h \
Index: configure.ac
===================================================================
--- configure.ac	(Revision 2056)
+++ configure.ac	(Arbeitskopie)
@@ -597,7 +597,6 @@
 AC_CONFIG_FILES([Makefile 
 src/Makefile 
 src/3rdParty/Makefile 
-src/3rdParty/Pivy/Makefile 
 src/Base/Makefile 
 src/App/Makefile 
 src/Gui/Makefile 
